src/services/uniswap/UniswapPriceService.ts

```diff
@@ -22,8 +22,8 @@ export class UniswapPriceService {
 
   async getLatestPrice(symbol: string, from: number, to: number): Promise<number | undefined> {
     return await this.priceRepository.getLatestPrice({
-      source: this.SOURCE,
-      symbol,
+      source: UniswapPriceService.SOURCE,
+      symbol: symbol.toUpperCase(),
```

Comment:
same question https://github.com/umbrella-network/pegasus/pull/258/files#r1034678851

Reply:
I will remove the uppercase here since we are saving in uppercase